examples/grafana/httproute_basic/README.md

```diff
@@ -0,0 +1,36 @@
+# Basic HTTPRoute Example
+
+This example demonstrates basic HTTPRoute configuration for Grafana using Gateway API.
+
+## Prerequisites
+
+- Kubernetes cluster with Gateway API CRDs installed
+- A Gateway resource named `my-gateway` in the `default` namespace
+- grafana-operator installed
+
+## Install Gateway API CRDs
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.1/standard-install.yaml
+```
+
+## Deploy
+
+```bash
+kubectl apply -f resources.yaml
+```
+
+## Configuration
+
+The example creates:
+- Grafana instance with basic HTTP configuration
+- HTTPRoute resource pointing to `my-gateway`
+- Hostname: `grafana.example.com`
+
+## Access
+
+Once deployed, Grafana will be accessible at `http://grafana.example.com` (depending on your Gateway configuration).
+
+Default credentials:
+- Username: `root`
+- Password: `secret`
```

Comment:
Let's not include any credentials here. - It's not consistent with other docs and is likely to get out of sync with the manifests at some point.

Reply:
Added frontmatter (title/linkTitle) and readfile shortcode for manifest inclusion.